src/reanimated2/globals.d.ts

```diff
@@ -100,4 +100,5 @@ declare global {
   var LayoutAnimationsManager: LayoutAnimationsManager;
   var UpdatePropsManager: UpdatePropsManager;
   var ProgressTransitionRegister: ProgressTransitionRegister;
+  var _jsPropsUpdater: (viewTag: number, props: unknown) => void;
```

Comment:
According to our convention, we use single leading underscore to represent JSI bindings, but this function is actually written in pure JavaScript. Can we rename it? Also, can we rename it to `updateJSProps` instead of `updater`? Finally, can we type `props` accordingly instead of using `unknown`?

Reply:
> "can we type props accordingly instead of using unknown?" 

The best what I can do is `Record` because there is no rules (or I don't know about them) how JS props can looks like 🤔